app/controllers/api/v1/books_controller.rb

```diff
@@ -0,0 +1,15 @@
+module Api
+  module V1
+    class BooksController < ApiController
+      def index
+        @books = Book.all
+        render json: @books
```

Comment:
are you sure this is returning the JSON that is specified in Apiary ?

Reply:
Yes. It is returning a JSON with the same format as Apiary